### PR TITLE
doc: move code of conduct to CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by MylesBorins**
It is currently in the GOVERNAMCE.md file. It doesn't seem like a good
idea to assume people are going to look for it there.

### CODE REVIEW

File: CODE_OF_CONDCUCT.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Code of Conduct
+
+The [Node.js Code of Conduct][] applies to this Team.
+
+[Node.js Code of Conduct]: https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md
+[Node.js Moderation Policy]: https://github.com/nodejs/admin/blob/master/Moderation-Policy.md
```
[the remark this answers is not in the document]
Comment (benjamingr) [reply CODE_OF_CONDCUCT.md]: Why not?

**Comment by giltayar** [comment]
typo in name: it should be CODE_OF_CONDUCT, not CODE_OF_CONDCUCT.

File: CODE_OF_CONDCUCT.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (targos) [reply CODE_OF_CONDCUCT.md]: The link is declared but unused. It doesn't show up in the rendered view: https://github.com/nodejs/modules/blob/3e17e4d7eb1d8344331935e4efdbd6811f78250f/CODE_OF_CONDUCT.md

**Comment by MylesBorins** [comment]
@targos good catch, I've pushed a new commit that should fix both points you raised

**Comment by manekinekko** [comment]
LGTM

**Pull request merged**